browser_history/utils.py

```diff
@@ -37,3 +37,24 @@ def get_platform():
     if system == "Windows":
         return Platform.WINDOWS
     raise NotImplementedError(f"Platform {system} is not supported yet")
+
+def get_history():
+    """This method is used to obtain browser histories of all available and supported
+    browsers for the system platform.
+
+    :return: Object of class :py:class:`browser_history.generic.Outputs` with the
+    data member entries set to list(tuple(:py:class:`datetime.datetime`, str))
+    
+    :rtype: :py:class:`browser_history.generic.Outputs`
+    """
+    output_object = generic.Outputs()
+    subclasses = generic.Browser.__subclasses__()
+    try:
+        for browser_class in subclasses:
+            browser_object = browser_class()
+            browser_output_object = browser_object.fetch()
+            output_object.entries.extend(browser_output_object.get())
+    except AssertionError:
+        pass
```

Comment:
- This will exit silently even if one browser is not supported i.e., it will stop trying after it finds an unsupported browser. The try-except should be inside the loop to prevent this.
 - Instead of silently ignoring unsupported browsers, it would be useful in debugging if it logs the unsupported browsers instead. [`logging.info`](https://docs.python.org/3/library/logging.html#logging.info) can be used for this.